aws/sdk/examples/ebs/src/bin/get-snapshot-state.rs

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
can you add a comment calling out that this actually uses the ec2 API?

Reply:
Done.